### DISCUSSION TITLE
[Application Development] transaction from op stack is not processing

### DISCUSSION THREAD

**Discussion by rohit98888** [ethereum-optimism/developers#96] [Application Development]
### Issue Description

Deposit on op-stack is happening but when I try to withdraw, metamask is showing tx as pending. Also when i try to deploy the contract with remix it shows me the error after a while
```
"
creation of Dai errored: Error occured: Transaction started at 17791 but was not mined within 53 blocks. Please make sure your transaction was properly sent and there no pervious pending transaction for the same account. However, be aware that it might still be mined!
	Transaction Hash: 0xd2e28bbb8d00a0386dc43195c9a827df9994bc2f97e5dddc8fdc9e6313c1d4c2.
"
```

**Comment by opfocus** [comment]
Are you providing enough basefee?

**Comment by sbvegan** [comment]
Can you provide more context:

- Are you running your own testnet op-stack chain?
- Are all of the components running?
- Are you seeing any logs that might help us debug?

**Reply by sbvegan** [reply]
Hey @rohit98888, I'm going to close this for now. If you can provide more information, we can reopen this thread and debug the issue further.
